### ISSUE TITLE
We need to run `npm rebuild` due to arch issues

### ISSUE THREAD

**Issue by theodorDiaconu** [zodern/mup-aws-beanstalk#36]
```Error: Failed to load gRPC binary module because it was not installed for the current system
Expected directory: node-v57-linux-x64-glibc
Found: [node-v46-darwin-x64-unknown, node-v48-linux-x64-glibc, node-v57-darwin-x64-unknown]
This problem can often be fixed by running "npm rebuild" on the current system
Original error: Cannot find module '/var/app/current/programs/server/npm/node_modules/grpc/src/node/extension_binary/node-v57-linux-x64-glibc/grpc_node.node'```

Awesome package!

**Comment by joe-santy** [comment]
See [This Solution](https://stackoverflow.com/questions/50479816/node-v57-linux-x64-glibc-grpc-node-node-missing-when-using-clasp-on-linux/50493985)

**Comment by zodern** [comment]
This is fixed in 0.7: #142
